test/Identity.FunctionalTests/Extensions/HtmlAssert.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using AngleSharp.Dom;
+using AngleSharp.Dom.Html;
+using AngleSharp.Parser.Html;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Identity.FunctionalTests
+{
+    public class HtmlAssert
```

Comment:
Did you consider just subclassing from `Xunit.Assert`? https://github.com/aspnet/Razor/blob/dev/test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/Assert.cs#L12